truffle/src/main/ruby/core/type.rb

```diff
@@ -419,6 +419,22 @@ def self.compatible_encoding(a, b)
       enc
     end
 
+    # similar to rb_inspect
+    def self.object_inspect(val)
```

Comment:
`Rubinius::Type.object_*` are usually for the original version of these methods on Object/Kernel, like `class`, etc.
Maybe just `inspect` here is fine? (so we just remove the `rb_` prefix)